use_cases/eluc/data/conversion.py

```diff
@@ -17,7 +17,7 @@
     "RUS": 643,
     "N": 578,
     "F": 250,
-    "J": 388,
+    # "J": 388,
```

Comment:
What was wrong with "J"?

Reply:
There are multiple countries with J in the converter I found. At some point the country code to name conversion scheme may have to be overhauled because it doesn't work for some countries.